app/Nova/Dashboards/Main.php

```diff
@@ -4,20 +4,47 @@
 
 namespace App\Nova\Dashboards;
 
-use Laravel\Nova\Cards\Help;
+use App\Nova\Metrics\AverageDaysToApproveExpenseReport;
+use App\Nova\Metrics\AverageDaysToCreateExpenseReport;
+use App\Nova\Metrics\AverageDaysToPayExpenseReport;
+use App\Nova\Metrics\DocuSignEnvelopesMissingExpenseReports;
+use App\Nova\Metrics\ExpensePaymentsPendingReconciliation;
+use App\Nova\Metrics\ExpenseReportsPendingApproval;
+use App\Nova\Metrics\ExpenseReportsPendingPayment;
 use Laravel\Nova\Dashboards\Main as Dashboard;
 
 class Main extends Dashboard
 {
+    /**
+     * Get the displayable name of the dashboard.
+     */
+    public function name(): string
+    {
+        return 'Home';
+    }
+
     /**
      * Get the cards for the dashboard.
      *
-     * @return array<\Laravel\Nova\Cards\Help>
+     * @return array<\Laravel\Nova\Card>
      */
     public function cards(): array
     {
         return [
-            new Help(),
+            DocuSignEnvelopesMissingExpenseReports::make()
+                ->width('1/4'),
+            ExpenseReportsPendingApproval::make()
+                ->width('1/4'),
+            ExpenseReportsPendingPayment::make()
+                ->width('1/4'),
```

Comment:
I thought about combining these, so there would just be one card showing expense reports that were created and not paid